components/posts/posts.php

```diff
@@ -29,29 +29,34 @@ function mm_posts( $args ) {
 		'taxonomy'            => '',
 		'term'                => '',
 		'heading_level'       => 'h1',
-		'per_page'            => 10,
+		'per_page'            => '',
```

Comment:
I think this default should stay here. Specifying all the defaults in one place is the way to go.